### ISSUE TITLE
Localhost Refused to Connect

### ISSUE THREAD

**Issue by SrtoGoge** [molnarmark/sublime-live-server#12]
I've setted all the paths correctly, as you can see below.

`{
	"node_executable_path": "C:/Program Files/nodejs/node.exe",
	"global_node_modules_path": "C:/Users/(My user)/AppData/Roaming/npm/node_modules"
}`

And it shows the _"Live Server running on port 8080"_ message. The problem comes when trying to run the _Open in Browser_ command, as it shows **Localhost refused to connect**.

I've seen that another user had the same [problem](https://github.com/molnarmark/sublime-live-server/issues/2) before, but I can't find a way to fix it. Running on Windows 10, using the latest version of Sublime Text, the package seems to be the latest, but even if all looks right, this is what I get. (It is on spanish, but I think I've been clear enough)
![image](https://github.com/molnarmark/sublime-live-server/assets/62667318/7105e532-618e-4c78-89ae-484dde19c9a4)

**Comment by molnarmark** [comment]
Hi,

I assume you have the [Live Server](https://www.npmjs.com/package/live-server) package installed globally?

If not, that is a requirement.
If you do, can you try running the live-server command from the command line / powershell, just to see if it fires up?

**Comment by Sleepy-gogo** [comment]
Indeed!

This is the weird part, I tried using the package itself and sure enough, it works

![image](https://github.com/molnarmark/sublime-live-server/assets/62667318/05e1abe9-3aca-4f83-9820-73d6f7a76fec)

But when trying to launch a server from the extension it's when it doesn't work! So naturally I've just been using the package, but tried posting this anyway so it may get fixed for future users

**Comment by Dashrath158** [comment]
Hey, I have the same issue so just like you I am also using it directly from the cmd but mine takes so much time to open folders, did you encounter something similar? Can it be a hardware issue?

**Comment by Sleepy-gogo** [comment]
Maybe it is hardware issue, my machine is actually really outdated, and runs slowly for certain tasks, so, it wouldn't be a surprise if that's the case with this.

I'm gonna ask one of my buddies with a better pc to try the extension and see if they have the same issue, but I really don't see why it should be a problem tho.

**Comment by molnarmark** [comment]
Maybe it also could be permission issues? Node being on drive A, sublime being on drive B, etc. Windows can be problematic with cases like those.

I don't use Windows for development, but when I tested the extension I had no issues myself, but of course I see some people reporting issues like this, so something is definitely up - I'm unable to reproduce the issue though sadly. 😦

**Comment by Dashrath158** [comment]
So on my end I checked the firewall settings they were alright, port 8080 was also free and no process was using it, disabled controlled foldered access, I had apache running in background turned that off too still the problem persists. My node and Sublime are also on same drive in ssd.

**Comment by Sleepy-gogo** [comment]
Same here, tried looking into the firewall, and running processes long ago, and I know that everything's alright.

I actually asked one of my friends who has a better pc and everything seems fine on his end.
![image](https://github.com/molnarmark/sublime-live-server/assets/62667318/79305330-d16c-46b0-a6c3-31257055a8d0)

For once, it actually may be hardware issues. I don't know what else it could be.

**Comment by molnarmark** [comment]
Yeah, this is the issue. It only happens for some, thus I can't reproduce it. It also only seems to happen on Windows.
